### PR TITLE
OSDOCS-2647: Adding release note for RHOCS AMI for AWS GovCloud

### PR DESCRIPTION
**Pull request by mjpytlak**
CP to 4.10

This PR is the release note for https://issues.redhat.com/browse/OSDOCS-2647.

Doc preview

[Installing a cluster into an Amazon Web Services GovCloud region](https://deploy-preview-40352--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-10-release-notes#ocp-4-10-installation-and-upgrade-aws-ami)

### CODE REVIEW

**Comment by mjpytlak** [comment]
@yunjiang29 @miabbott Ahead of a preliminary drop of the 4.10 release notes early next week, requesting QE review. A link to the revised installation instructions for GovCloud will be added as part of GA readiness activities.

**Comment by yunjiang29** [comment]
Thanks @mjpytlak for the updates, the text part LGTM.

**Comment by yunjiang29** [comment]
Hi @mjpytlak, I got a question, since we have rhcos images on AWS GovCloud, it looks like that the section [Uploading a custom RHCOS AMI in AWS](https://docs.openshift.com/container-platform/4.9/installing/installing_aws/installing-aws-government-region.html#installation-aws-upload-custom-rhcos-ami_installing-aws-government-region) need to be updated, it's for the secret regions only now, is there any JIRA issue or PR to track this? Or we are going to separate the Gov and Secret regions into 2 sections in the document.

**Comment by mjpytlak** [comment]
@yunjiang29 Yes. I have split the GovCloud and Secret regions into two separate docs. I had meant to include those references in this PR. Each new section is out to SME review. Once I  have approval, I will be forwarding to you. 

- GovCloud [1]
- Secret [2]

[1] https://github.com/openshift/openshift-docs/pull/39614
[2] https://github.com/openshift/openshift-docs/pull/39769

File: release_notes/ocp-4-10-release-notes.adoc [hunk 1]
```diff
@@ -48,6 +48,12 @@ This release adds improvements related to the following components and concepts.
 [id="ocp-4-10-installation-and-upgrade"]
 === Installation and upgrade
 
+[id="ocp-4-10-installation-and-upgrade-aws-ami"]
+==== Installing a cluster into an Amazon Web Services GovCloud region
+
+{op-system-first} Amazon Machine Images (AMIs) are now available for AWS GovCloud regions. The availability of these AMIs improves the installation process, as you are no longer required to upload a custom RHCOS AMI to deploy a cluster.
```
Comment (bergerhoffer) [inline release_notes/ocp-4-10-release-notes.adoc:54]: as -> because (per IBM style guide)

**Review by bergerhoffer — COMMENTED** [review]
Just one comment (not including the part about adding the xref once it's available), otherwise LGTM!

**Comment by bergerhoffer** [comment]
LGTM merging

**Pull request merged**